packages/documentation/pages/usage/components/field-inline-edit.vue

```diff
@@ -0,0 +1,210 @@
+								{ key: 'isDisabled', label: 'isDisabled' },
+								{ key: 'isReadonly', label: 'isReadonly' },
+								{ key: 'isLoading', label: 'isLoading' },
+								{ key: 'isMultiline', label: 'isMultiline' },
+								{ key: 'isOptional', label: 'isOptional' },
+							]"
+							type="switch"
+						/>
+						<KtFieldText formKey="placeholder" isOptional label="placeholder" />
+						<KtFieldNumber formKey="tabIndex" isOptional label="tabIndex" />
+						<KtFieldText formKey="dataTest" isOptional label="dataTest" />
+					</div>
+
+					<div>
+						<h4>Texts</h4>
+						<KtFieldText formKey="label" isOptional label="label" />
+						<KtFieldText
+							formKey="helpDescription"
+							isOptional
+							label="helpDescription"
+						/>
+						<div class="field-row">
+							<KtFieldText formKey="helpText" isOptional label="helpText" />
+						</div>
+					</div>
+				</div>
+			</KtForm>
+		</div>
+	</div>
+</template>
+
+<script lang="ts">
+import {
+	Kotti,
+	KtFieldInlineEdit,
+	KtForm,
+	KtFieldSingleSelect,
+} from '@3yourmind/kotti-ui'
+import { KottiFieldInlineEdit } from '@3yourmind/kotti-ui/dist/esm/kotti-field-inline-edit/types'
+import { defineComponent, ref, computed } from '@vue/composition-api'
+
+import ComponentInfo from '~/components/ComponentInfo.vue'
+
+export default defineComponent({
+	name: 'DocumentationPageUsageComponentsFieldInlineEdit',
+	components: {
+		ComponentInfo,
+		KtFieldInlineEdit,
+		KtForm,
+		KtFieldSingleSelect,
+	},
+	setup() {
+		const getInitialValue = () => ({ confirmedValue: null })
+
+		const settings = ref<{
+			booleanFlags: {
+				hideValidation: Kotti.FieldToggle.Value
+				isDisabled: Kotti.FieldToggle.Value
+				isReadonly: Kotti.FieldToggle.Value
+				isLoading: Kotti.FieldToggle.Value
+				isMultiline: Kotti.FieldToggle.Value
+				isOptional: Kotti.FieldToggle.Value
+			}
+			dataTest: Kotti.FieldText.Value
+			helpDescription: Kotti.FieldText.Value
+			helpText: Kotti.FieldText.Value
+			label: Kotti.FieldText.Value
+			placeholder: Kotti.FieldText.Value
+			preventConfirmationOn: Kotti.FieldInlineEdit.ConfirmationValidation
+			tabIndex: Kotti.FieldNumber.Value
+			textStyle: Kotti.FieldInlineEdit.TextStyle | null
+			validation: Kotti.Field.Validation.Result['type']
+		}>({
+			booleanFlags: {
+				hideValidation: false,
+				isDisabled: false,
+				isReadonly: false,
+				isLoading: false,
+				isMultiline: true,
+				isOptional: true,
+			},
+			dataTest: null,
+			helpDescription: null,
+			helpText: null,
+			label: 'Label',
+			placeholder: null,
+			preventConfirmationOn: KottiFieldInlineEdit.ConfirmationValidation.NEVER,
+			tabIndex: null,
+			textStyle: null,
+			validation: 'empty',
+		})
+
+		const formValue = ref<{ confirmedValue: KottiFieldInlineEdit.Value }>(
+			getInitialValue(),
+		)
+		const confirmedValue = computed(() => formValue.value.confirmedValue)
+
+		// const confirmedValue = ref<KottiFieldInlineEdit.Value>(null)
```

Comment:
Is this needed?

Reply:
it's for reference to use outside a form, can keep, can delete (see comments by me on this file)